packages/app/client/src/components/projects/create/components/forms.js

```diff
@@ -499,43 +514,37 @@ export const FirstProjectForm = props => {
                     />
                   </div>
                 )}
+                <FormHelperText className={classes.errorMessage}>
+                  {!!(touched.attachment && errors.attachment) && !oldPicture
+                    ? !!touched.attachment && errors.attachment
+                    : null}
+                </FormHelperText>
               </Grid>
               <Grid item xs={12}>
-                <InputLabel
-                  className={classes.inputLabel}
-                  shrink
-                  htmlFor="projectName"
-                >
-                  Project name
-                </InputLabel>
                 <TextField
                   className={classes.formElement}
                   id="projectName"
                   variant="outlined"
                   name="projectName"
+                  htmlFor="projectName"
+                  label="Project name"
                   onChange={handleChange}
                   onBlur={handleBlur}
                   onKeyPress={disableEnterSubmit}
                   error={!!(touched.projectName && errors.projectName)}
-                  errorMessage={!!values.projectName && errors.projectName}
+                  errorMessage={!!touched.projectName && errors.projectName}
```

Comment:
Hey, regarding the Formik docs (https://formik.org/docs/api/formik#touched--field-string-boolean-) the touched[fieldName] is a boolean value so `!!touched.projectName` is equals to `touched.projectName`